datasets/sentinel-s2-l2a-mosaic-120.yaml

```diff
@@ -0,0 +1,41 @@
+Name: Sentinel-2 L2A 120m Mosaic
+Description: |
+  The [Sentinel-2 mission](https://sentinel.esa.int/web/sentinel/missions/sentinel-2) is
```

Comment:
@gmilcinski Could you add some callout here to what the 120m product is? Right now this description is just about S2 data.

Reply:
Darn, the copy-paste. Sorry about that. I believe I upated the file, but not really sure as, as you might remember, I am the worse programmer ever.  
This is what I tried to add there:  
Sentinel-2 L2A 120m mosaic is a derived product, which contains best pixel values for 10-daily periods, modelled by removing the cloudy pixels and then performing interpolation among remaining values. As there are some parts of the world, which have lengthy cloudy periods, clouds might be remaining in some parts. The actual modelling script is available [here](https://sentinel-hub.github.io/custom-scripts/sentinel-2/interpolated_time_series/).